src/wayfinding/types/index.ts

```diff
@@ -69,6 +69,17 @@ export interface MapViewRef {
    * This action will have the same effect as the user searching in the searchbar.
    */
   search: (searchFilter: SearchFilter) => void;
+
+  /**
+   * Selects a point saved as find my car on the map.
```

Comment:
A min pareceme que queda mellor asi, como o ves ?
Selects the point where your car is located at.

To use it, you will need the 'Find my car' feature enabled.

Reply:
pois non está de máis meter eso, engádollo a ambos :)